modules/ROOT/pages/to-find-info.adoc

```diff
@@ -30,54 +13,131 @@ Exchange Search looks for terms in:
 * The content of the asset portal pages
 * xref:to-find-info.adoc#searching-api-specifications[Properties defined in RAML and OAS API specifications]
 
-You can combine search terms with the filters listed in All Types.
+In Exchange you can enter search text to search for assets, click *All Types* to set a filter for specific asset types, or combine search text with the filters listed in the type menu.
+
+<<index.adoc#content-types,Content Types>> lists the different types of asset.
+
+You can also search by clicking category or tag elements. This creates a filtered search you can use to find similar assets.
+
+The original Exchange search experience is avilable in Anypoint Platform Private Cloud Edition, MuleSoft Government Cloud, the US cloud, and the EU cloud. The new Exchange search experience is available in the US cloud and the EU cloud.
```

Comment:
Typo for available on line 22. Can we perhaps add a screenshot that introduces the "new search"? Remember, this is something the customers will still have to opt-into, so adding a screenshot will help in creating that clarity..

Reply:
Fixed that typo! Also added a screenshot of the new search.